### ISSUE TITLE
Throttle not idle and timer starting on x20

### ISSUE THREAD

**Issue by hancop** [FrSkyRC/ETHOS-Feedback-Community#1974]
I have 2 issues that I think are related. My throttle stick is at lowest position and shows at -100 percent on outputs screen for both the channel and mixer, but when I turn on the transmitter I get a warning message that throttle is not idle. I also have my timer set to to trigger from absolute (which means should run when not idle) and it starts right away even with throttle stick in what should be the idle position. It seems to me that even with my throttle stick in the idle position (all the way down) the transmitter still thinks it is not at idle. Does anyone have any idea why this might be ?

Thanks

**Comment by 4qr1hu9n7** [comment]
Calibrate the transmitter sticks, sliders, and pots. See if that helps. I periodically recalibrate the controls, especially after an update.

**Comment by hancop** [comment]
Did that, still does not work 😕

**Comment by MIKE-DAILEY** [comment]
Not sure about the counter, but what percentage is set for the throttle on the Checklist screen.  Should be -95.

**Comment by hancop** [comment]
Yes that fixed by checklist issue thanks. Still have an issue with throttle setup and have some more info. My throttle range goes from -100 to +100. I hooked it up to my esc and motor and the motor does not even start till I get the throttle stick To -10 percent, so,almost in the middle. I did calibrate the esc properly to let it determine max and mine throttle points as per its instructions. No idea how to set this up properly so motor starts the second it moves off the bottom position

Thanks
